Error on calculation scores - fitting
General_Descriptors Graph_based_Signatures Fragments # NoScaling # NoFeatureSelection # NeuroNet 1 identity adam adaptive 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNet <n_neurons>#<n_neurons> tanh lbfgs adaptive 250 0.07
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores Fragments # MaxAbsScaler # NoFeatureSelection # NeuroNet <n_neurons>#<n_neurons>#<n_neurons>#<n_neurons> identity sgd constant 3500 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Toxicophores Fragments # MinMaxScaler # NoFeatureSelection # NeuroNet <n_neurons>#<n_neurons>#<n_neurons> identity adam invscaling 3500 0.08
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet <n_neurons>#<n_neurons> identity sgd constant 1000 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # MinMaxScaler # NoFeatureSelection # NeuroNet <n_neurons>#<n_neurons>#<n_neurons> identity adam invscaling 3500 0.08
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Toxicophores Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet <n_neurons>#<n_neurons> identity sgd constant 1000 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # MinMaxScaler # NoFeatureSelection # NeuroNet <n_neurons>#<n_neurons> identity sgd constant 1000 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Toxicophores Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet <n_neurons>#<n_neurons>#<n_neurons> identity adam invscaling 3500 0.08
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on pipeline - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet <n_neurons>#<n_neurons> identity sgd constant 1000 0.06
'NoneType' object has no attribute 'fit'

Error on calculation scores - fitting
Graph_based_Signatures Toxicophores Fragments # NoScaling # NoFeatureSelection # NeuroNet 1 # identity adam adaptive 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # Normalizer l2 # NoFeatureSelection # NeuroNet 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNet 5 # 25 # 5 # logistic sgd invscaling 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler True False # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # Normalizer l2 # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # RobustScaler True False # NoFeatureSelection # NeuroNet 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler True False # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on pipeline - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
'NoneType' object has no attribute 'fit'

Error on calculation scores - fitting
Graph_based_Signatures Toxicophores Fragments # NoScaling # NoFeatureSelection # NeuroNet 1 # identity adam adaptive 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # Normalizer l2 # NoFeatureSelection # NeuroNet 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNet 5 # 25 # 5 # logistic sgd invscaling 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler True False # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # Normalizer l2 # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # RobustScaler True False # NoFeatureSelection # NeuroNet 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler True False # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on pipeline - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
'NoneType' object has no attribute 'fit'

Error on calculation scores - fitting
Graph_based_Signatures Toxicophores Fragments # NoScaling # NoFeatureSelection # NeuroNet 1 # identity adam adaptive 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNet 5 # 25 # 5 # logistic sgd invscaling 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # Normalizer l2 # NoFeatureSelection # NeuroNet 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler True False # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # Normalizer l2 # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # RobustScaler True False # NoFeatureSelection # NeuroNet 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler True False # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on pipeline - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
'NoneType' object has no attribute 'fit'

Error on calculation scores - fitting
Graph_based_Signatures Toxicophores Fragments # NoScaling # NoFeatureSelection # NeuroNet 1 # identity adam adaptive 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # Normalizer l2 # NoFeatureSelection # NeuroNet 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNet 5 # 25 # 5 # logistic sgd invscaling 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler True False # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # Normalizer l2 # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # RobustScaler True False # NoFeatureSelection # NeuroNet 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler True False # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on pipeline - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNet 2 # relu lbfgs invscaling 1000 0.09
'NoneType' object has no attribute 'fit'

Error on calculation scores - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Graph_based_Signatures Toxicophores Fragments # NoScaling # NoFeatureSelection # NeuroNets 1 # identity adam adaptive 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # Normalizer l2 # NoFeatureSelection # NeuroNets 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNets 5 # 25 # 5 # logistic sgd invscaling 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler True False # NoFeatureSelection # NeuroNets 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # RobustScaler True False # NoFeatureSelection # NeuroNets 750 # 250 # identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # Normalizer l2 # NoFeatureSelection # NeuroNets 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler True False # NoFeatureSelection # NeuroNets 2 # relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 10 # 500 # relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on pipeline - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 2 # relu lbfgs invscaling 1000 0.09
'NoneType' object has no attribute 'fit'

Error on calculation scores - fitting
Graph_based_Signatures Toxicophores Fragments # NoScaling # NoFeatureSelection # NeuroNets 1 identity adam adaptive 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # Normalizer l2 # NoFeatureSelection # NeuroNets 750 250 identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 2 relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNets 5 25 5 logistic sgd invscaling 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler True False # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # Normalizer l2 # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # RobustScaler True False # NoFeatureSelection # NeuroNets 750 250 identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler True False # NoFeatureSelection # NeuroNets 2 relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on pipeline - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 2 relu lbfgs invscaling 1000 0.09
'NoneType' object has no attribute 'fit'

Error on calculation scores - fitting
Graph_based_Signatures Toxicophores Fragments # NoScaling # NoFeatureSelection # NeuroNets 1 identity adam adaptive 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNets 5 25 5 logistic sgd invscaling 10 0.06
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 2 relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # Normalizer l2 # NoFeatureSelection # NeuroNets 750 250 identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler True False # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # Normalizer l2 # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # RobustScaler True False # NoFeatureSelection # NeuroNets 750 250 identity lbfgs adaptive 1000 0.05
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
Fragments # RobustScaler True False # NoFeatureSelection # NeuroNets 2 relu lbfgs invscaling 1000 0.09
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001
The 'estimator' parameter of cross_val_score must be an object implementing 'fit'. Got None instead.

Error on pipeline - fitting
Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 2 relu lbfgs invscaling 1000 0.09
'NoneType' object has no attribute 'fit'

Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNets 5 25 5 logistic sgd invscaling 10 0.06

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # Normalizer l2 # NoFeatureSelection # NeuroNets 750 250 identity lbfgs adaptive 1000 0.05

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNets 5 25 5 logistic sgd invscaling 10 0.06

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler True False # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # Normalizer l2 # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # RobustScaler True False # NoFeatureSelection # NeuroNets 750 250 identity lbfgs adaptive 1000 0.05

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler True False # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # Normalizer l2 # NoFeatureSelection # NeuroNets 750 250 identity lbfgs adaptive 1000 0.05

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
Advanced_Descriptors Graph_based_Signatures Toxicophores # Normalizer max # NoFeatureSelection # NeuroNets 5 25 5 logistic sgd invscaling 10 0.06

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # Normalizer l2 # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures # RobustScaler True False # NoFeatureSelection # NeuroNets 750 250 identity lbfgs adaptive 1000 0.05

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Fragments # RobustScaler False True # NoFeatureSelection # NeuroNets 10 500 relu adam invscaling 2500 0.0000000001

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors Graph_based_Signatures Toxicophores Fragments # NoScaling # NoFeatureSelection # NeuroNets 500 5 identity adam adaptive 100 0.06

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Fragments # MaxAbsScaler # NoFeatureSelection # NeuroNets 1000 250 1000 relu sgd adaptive 250 0.05

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Graph_based_Signatures Toxicophores Fragments # MaxAbsScaler # NoFeatureSelection # NeuroNets 3 50 relu adam adaptive 250 0.07

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Toxicophores Fragments # MaxAbsScaler # NoFeatureSelection # NeuroNets 1000 250 1000 relu sgd adaptive 250 0.05

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


Error on calculation scores - fitting
General_Descriptors Advanced_Descriptors # RobustScaler True True # NoFeatureSelection # NeuroNets 1000 250 1000 relu sgd adaptive 250 0.05

All the 5 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
5 fits failed with the following error:
Traceback (most recent call last):
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 895, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/base.py", line 1474, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 752, in fit
    return self._fit(X, y, incremental=False)
  File "/home/alexgcsa/anaconda3/envs/autoadmet/lib/python3.10/site-packages/sklearn/neural_network/_multilayer_perceptron.py", line 434, in _fit
    if np.any(np.array(hidden_layer_sizes) <= 0):
numpy.core._exceptions._UFuncNoLoopError: ufunc 'less_equal' did not contain a loop with signature matching types (<class 'numpy.dtypes.StrDType'>, <class 'numpy.dtypes.Int64DType'>) -> <class 'numpy.dtypes.BoolDType'>


